src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -164,10 +165,16 @@ local function writeHighlighted(sLine)
             tryWrite(sLine, "^\'\'", stringColour) or
             tryWrite(sLine, "^\'.-[^\\]\'", stringColour) or
             tryWrite(sLine, "^%[%[.-%]%]", stringColour) or
-            tryWrite(sLine, "^[%w_]+", function(match)
+            tryWrite(sLine, "^[%w_%.]+", function(match)
                 if tKeywords[match] then
                     return keywordColour
                 end
+                if tFunctions[match] then
+                    return functionColour
+                end
+                if match == "true" or match == "false" then
```

Comment:
Maybe make `tKeywords` a mapping of string → colours, so one may do `return tKeywords[match] or textColour`, rather than this if-else chain.

Reply:
Sounds good.